src/ralph/data_importer/resources.py

```diff
@@ -28,12 +32,26 @@
 )
 from ralph.supports.models import BaseObjectsSupport, Support, SupportType
 
+RalphResourceMeta = type(
+    'RalphResourceMeta',
+    (
+        ImportForeignKeyMeta,
+        resources.ModelDeclarativeMetaclass,
+        resources.Resource
+    ),
+    {}
+)
+
 
 class DefaultResource(ImportForeignKeyMixin, resources.ModelResource):
     pass
 
 
-class AssetModelResource(ImportForeignKeyMixin, resources.ModelResource):
+class AssetModelResource(
+    ImportForeignKeyMixin,
+    resources.ModelResource,
```

Comment:
instead of using metaclass here, what do you think about creating new class RalphModelResource that will use this metaclass and inherit from it here?